src/containers/documentation/swaggerPlugins/CurlForm.tsx

```diff
@@ -236,7 +243,8 @@ export class CurlForm extends React.Component<CurlFormProps, CurlFormState> {
   }
 
   public authParameterContainer(): JSX.Element {
-    if (Object.keys(this.props.operation.security[0]).includes('apikey')) {
+    const security = this.security() ?? [];
```

Comment:
This is a simple fix
```suggestion
    const security = this.security() ?? [{}];
```